core/collection-factory/api/src/main/java/org/eclipse/rdf4j/collection/factory/api/BindingSetEntry.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
```

Comment:
The format of the copyright headers has changed: we've introduce a machine-readable tag and put some additional whitespace in to make it easier to read:

```
/*******************************************************************************
 * Copyright (c) ${year} Eclipse RDF4J contributors.
 *
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *
 * SPDX-License-Identifier: BSD-3-Clause
 *******************************************************************************/
```

Reply:
I fixed them for the new files of this PR, there seem to be a few left over files that do not have the new SPDX header nor the one mentioning aduna.

```
for i in $(find . -name *.java -exec grep -L 'SPDX-License' {} \;);do if [[ ! $i =~ "generated" ]];then grep -L aduna $i;fi;done |wc -l
```